lib/signaling/participant.js

```diff
@@ -37,94 +34,97 @@ var states = {
  * @emits ParticipantSignaling#trackAdded
  * @emits ParticipantSignaling#trackRemoved
  */
-function ParticipantSignaling() {
-  StateMachine.call(this, 'connecting', states);
+class ParticipantSignaling extends StateMachine {
+  /**
+   * Construct a {@link ParticipantSignaling}.
+   */
+  constructor() {
+    super('connecting', states);
 
-  Object.defineProperties(this, {
-    _identity: {
-      writable: true,
-      value: null
-    },
-    _sid: {
-      writable: true,
-      value: null
-    },
-    identity: {
-      enumerable: true,
-      get: function() {
-        return this._identity;
+    Object.defineProperties(this, {
+      _identity: {
+        writable: true,
+        value: null
+      },
+      _sid: {
+        writable: true,
+        value: null
+      },
+      identity: {
+        enumerable: true,
+        get() {
+          return this._identity;
+        }
+      },
+      sid: {
```

Comment:
@markandrus Use a getter method?

Reply:
That's a good suggestion—maybe we can wait to do that in another PR, though? There are some other changes we could make that I haven't included here (for example, `async`/`await`).